### ISSUE TITLE
I train ResNet18 in Cifar100

### ISSUE THREAD

**Issue by dreamer121121** [VITA-Group/AugMax#3]
I train ResNet18 in Cifar100 with augmax and without augmax  but i find that i can got 77.5% accuracy without augmax ,while  i can only got 76.3% with augmax, can u give some explainations?

**Comment by htwang14** [comment]
Hi! What Lambda value did you use? In the paper, we mentioned that on CIFAR100 with ResNet18 we use Lambda=1.0.

**Comment by dreamer121121** [comment]
Thanks for ur reply, i set the lambda=1.0 then i got ur results in CIfar100. but, i have another question, how to choose the lambda when using the augmax in different datasets?

**Comment by dreamer121121** [comment]
> Hi! What `Lambda` and `steps` values did you use? In the paper, we mentioned that on CIFAR100 with ResNet18 we use Lambda=1.0. `steps` should always be `10` on CIFAR10/100 and Tiny ImageNet. I've updated README in [8ca00c0](https://github.com/VITA-Group/AugMax/commit/8ca00c04abb20cfd78d58b0dd8a2955176babb54) to reflect these details. Sorry for the confusion.
Thanks for ur reply, i set the lambda=1.0 then i got ur results in CIfar100. but, i have another question, how to choose the lambda when using the augmax in different datasets?

**Comment by htwang14** [comment]
My suggestion is to first try Lambda~10. Usually AugMax gets similar performance with different Lambda values. The experiment on CIFAR100 with ResNet18 is the only exception I encountered, where smaller Lambda is preferred. My assumption is that ResNet18 has limited model capacity, so adding too much regularizations (by using a large Lambda) make the model over-smooth and hurts performance.

**Comment by dreamer121121** [comment]
> My suggestion is to first try Lambda~10. Usually AugMax gets similar performance with different Lambda values. The experiment on CIFAR100 with ResNet18 is the only exception I encountered, where smaller Lambda is preferred. My assumption is that ResNet18 has limited model capacity, so adding too much regularizations (by using a large Lambda) make the model over-smooth and hurts performance.

I want to apply Augmax to semantic segmentation, can you give me some advise?

**Comment by htwang14** [comment]
I think AugMax is directly appliable on semantic segmentation tasks: just replace the classification loss with dense prediction loss (both the cross-entropy and the consistency loss should be replaced with a dense prediction version). As for benchmark datasets, you can use this one [https://github.com/bethgelab/robust-detection-benchmark](https://github.com/bethgelab/robust-detection-benchmark), which adds corruptions on segmentation/detection datasets. Although it's named as object detection, I think is can also be used for segmentation (at least Cityscapes and Pascal VOC have segmentation annotations).
